tools/pontoon/teams_projects.md

```diff
@@ -0,0 +1,76 @@
+
+![Team page for Italian](/assets/images/pontoon/team_project/team_page.png)
+
+The page header contains linguistic information about the current locale (plural form, writing direction, etc.), plus an indication of the overall status of completion and statistics.
+
+![Header of team page for Italian](/assets/images/pontoon/team_project/team_page_header.png)
+
+The [Teams page](https://pontoon.mozilla.org/teams/) will have similar information, but referring to Pontoon as a whole. For example: number of teams, team with most translations or missing strings, overall completion and statistics across projects and locales.
+
+There are 4 tabs available (highlighted in red), 5 if you’re an *admin* or *manager*.
+
+* **Projects**: a list of all projects enabled in Pontoon for this locale.
+* **Contributors**: a list of active contributors with their statistics, filterable by time (all time, 1/3/6/12 months).
+* **Bugs**: a list of open bugs for this locale, retrieved from [Bugzilla](https://bugzilla.mozilla.org/).
+* **Info**: information about the team. This is currently only editable by administrators.
+* **Permissions**: manage user permissions (see [this document](users.md#managing-permissions) for details).
+
+Hovering a project in the list will replace the progress bar with a set of detailed statistics (untranslated strings, missing strings, etc.). Note that all these numbers are links, so you can use them to open the project with a filter already enabled, for example to display only missing strings. You can click on the number of total strings to access *All Resources* (i.e. all strings in all files) for this project.
+
+![Project hover](/assets/images/pontoon/team_project/project_hover.png)
+
+### Requesting a project
+
+It’s possible to request a project from a locale’s *Team page*.
+
+![Request a project](/assets/images/pontoon/team_project/request_project.png)
+
+Click on *REQUEST MORE PROJECTS* (highlighted in red), select the project you want to add and then click *REQUEST NEW PROJECT* (you need to select at least one project for the button to be displayed).
+
+![Request a project](/assets/images/pontoon/team_project/request_project_selected.png)
+
+An email will be sent to Pontoon’s administrators, and the localization driver in charge of the project will act on the request. Please note that:
+* Some projects have a closed list of supported locales, meaning that you can’t request the project on Pontoon.
+* Some projects can be requested but eventually not enabled, because of practical restrictions related to the project itself (e.g. lack of support for the locale in iOS).
+
+## Project page
+
+Similar to the [Teams page](https://pontoon.mozilla.org/teams/), there’s also a landing [Projects page](https://pontoon.mozilla.org/projects/). From this page it’s possible to access each *Project page*, to see the overall translation status, or when was the last time that each locale worked on the project.
```

Comment:
The Projects page has already been introduced on top of the document.

Instead of this paragraph, we should introduce the Project page, similarly as we introduce Team and Localization pages:
1. How to access it.
2. Screenshot.
3. Header.
4. Tabs